app/views/application/_header.html.erb

```diff
@@ -1,16 +1,16 @@
 <nav class="header">
   <ul class="main-nav">
     <li>
-      <%= link_to t("families.index.home"), root_path %>
+      <%= link_to t("families.header.links.home"), root_path %>
```

Comment:
I think the key should be: "header.links.home", rather than being nested instead the `families` key

Reply:
👌